dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -118,7 +118,9 @@ def k5_public_map_index
     # will be to show workshops from all subjects when deep-dive isn't specified.
     # But until then, when deep-dive isn't specified, we only show 'Intro'
     # workshops.
-    conditions[:subject] = params['deep_dive_only'] ? 'Deep Dive' : 'Intro'
+    if params['deep_dive_only']
```

Comment:
Does this change the comment above it?

Reply:
Whoops, yes - it makes it irrelevant.  Removing!